standardise_ip.py

```diff
@@ -237,18 +237,21 @@ def processed_cell_values():
 
             yield (row_id, rawvalue, cidrized)
 
+    dictionary_of_rows_ips = {}
     for row_id, rawvalue, cidrized in processed_cell_values():
-        sheet.cell(column=columnnumber+1, row=row_id).value = str(cidrized)
+        #sheet.cell(column=columnnumber+1, row=row_id).value = str(cidrized)
+        dictionary_of_rows_ips[row_id] = map(str,cidrized)
+    return dictionary_of_rows_ips
+        
 
     print "There are %d bad ips need fixing." % bad_count
 
 # run()
 def run():
-    _, inputf, outputf, sheetname, columnnumber = sys.argv
+    _, inputf, sheetname, columnnumber = sys.argv
```

Comment:
Although we all do the opposite (constantly), it's good to break down your commits so that the name reflects what you've done; i.e. this change could have perfectly been committed separately.